### PR TITLE
FEC-10638 Add KS to playbackUrl for direct playbackContext url

### PR DESCRIPTION
**Pull request by chausov**
FEC-10638 

- Adding KS to playback URL logic
- Some extra fix for flavors as empty String "" 
- KalturaLiveStreamAdminEntry mapping

### CODE REVIEW

File: Sources/OVP/Parsers/OVPObjectMapper.swift [hunk 1]
```diff
@@ -29,7 +29,7 @@ class OVPObjectMapper: NSObject {
             switch name {
             case "KalturaMediaEntry":
                 return OVPEntry.self
-            case "KalturaLiveStreamEntry":
+            case "KalturaLiveStreamEntry", "KalturaLiveStreamAdminEntry":
```
Comment (x-NR-x) [inline Sources/OVP/Parsers/OVPObjectMapper.swift:32]: Need to see where this object type came from and if it's supposed to be parsed.
Checking...

Comment (x-NR-x) [reply Sources/OVP/Parsers/OVPObjectMapper.swift:32]: This is a legacy object done for QA Admin accounts.

File: Sources/OVP/Provider/OVPMediaProvider.swift [hunk 2]
```diff
@@ -406,7 +406,21 @@ public enum OVPMediaProviderError: PKError {
...
+                if sourceUrl.query == nil {
+                    let lastPathComponent = sourceUrl.lastPathComponent
+                    
+                    playURL = sourceUrl
+                        .deletingLastPathComponent()
+                        .appendingPathComponent("ks")
+                        .appendingPathComponent(ks)
+                        .appendingPathComponent(lastPathComponent)
+                } else {
+                    playURL = sourceUrl.appendingQueryComponent(key: "ks", value: ks)
```
Comment (x-NR-x) [inline Sources/OVP/Provider/OVPMediaProvider.swift:419]: Before merging need to set the dependency in the podspec to the Netkit once released.

Comment (chausov) [reply Sources/OVP/Provider/OVPMediaProvider.swift:419]: Done

**Review by x-NR-x — APPROVED** [review]

**Pull request merged**